fiftyone/utils/complex_yolo/model.py

```diff
@@ -1,50 +1,225 @@
 """
-Complex-YOLOv3 model adapted from https://github.com/ghimiredhikura/Complex-YOLOv3.
+Complex-YOLOv3 model adapted from
+https://github.com/ghimiredhikura/Complex-YOLOv3.
 
 | Copyright 2017-2022, Voxel51, Inc.
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
-from collections import OrderedDict
-from typing import Tuple, Union
-
 import math
 import numpy as np
-from shapely.geometry import Polygon
-import open3d as o3d
 
 import fiftyone.core.utils as fou
 
 fou.ensure_torch()
 import torch
-import torch.nn as nn
-import torch.nn.functional as nnF
-from torch.autograd import Variable
+from torch import nn
+import torch.nn.functional as F
 
+sg = fou.lazy_import(
```

Comment:
By convention, when we write utils that use modules that aren't included as default dependencies of `fiftyone` package, we use the pattern below so that the user will get an informative error message if they don't have `shapely` installed.

You can see this in use in other parts of the codebase.